packages/next/src/bin/next.ts

```diff
@@ -108,12 +108,11 @@ if (process.env.NODE_ENV) {
 ;(process.env as any).NODE_ENV = process.env.NODE_ENV || defaultEnv
 ;(process.env as any).NEXT_RUNTIME = 'nodejs'
 
-// Make sure commands gracefully respect termination signals (e.g. from Docker)
-// Allow the graceful termination to be manually configurable
-if (!process.env.NEXT_MANUAL_SIG_HANDLE && command !== 'dev') {
+if (command === 'build') {
```

Comment:
we still want to stop the CLI immediately if SIGTERM or SIGINT is used for building. There's also a test for this already that was breaking if we remove these handlers for all commands.